frontend/src/components/UserDetails.vue

```diff
@@ -93,6 +93,13 @@
               {{ identity.identityProvider | formatIdentityProvider }} [{{ identity.userName }}]
             </li>
           </ul>
+          <br/><br/>
+          <label for="all-roles">Application Roles</label>
+          <ul id="all-roles" style="margin-top: 5px; list-style: square">
+              <li v-for="client in user.allRoles" :key="client.clientName">
+                {{client.clientName}} [<span v-for="role in client.effectiveRoles" :key="role.id">{{role.name}} </span>]
```

Comment:
I was trying to figure out how to put a comma in between and came up with this:

`{{client.clientName}} [{{client.effectiveRoles.map(role => role.name).join(", ")}}]`